src/Microsoft.ML.OnnxTransformer/OnnxUtils.cs

```diff
@@ -348,7 +347,7 @@ public static OnnxModel CreateFromBytes(byte[] modelBytes)
         public static OnnxModel CreateFromBytes(byte[] modelBytes, int? gpuDeviceId = null, bool fallbackToCpu = false,
             IDictionary<string, int[]> shapeDictionary = null)
         {
-            var tempModelFile = Path.GetTempFileName();
+            var tempModelFile = Path.Combine(MLContext.TempFilePath, Path.GetRandomFileName());
```

Comment:
>Path.Combine(MLContext.TempFilePath, Path.GetRandomFileName()); [](http://example.com/codeflow?start=32&length=63)

I am seeing this is repeated. would make sense to have a property in MLContext to return the complete random file path?

Reply:
If we add this, it should be `internal`. We don't want to expose it publicly.

Reply:
Its a new random file path each time, so we probably wouldn't want it to be a property. I could add an internal method that returned that though.